crates/test_utils/README.md

```diff
@@ -1,13 +1,52 @@
-# Test Runner
+# Test Utils
 
-The main part of running tests through `newman`.
+The heart of `newman-dir` and `UI-tests`
 
-# Usage
+## Usage
 
-- Make sure you that you've the postman collection for the connector available in the `postman` dir with the name `<connector_name>.postman_collection.json`
+- Make sure you that you've **do not** have the official newman installed but rather `newman-dir`
+  - `newman-dir` can be installed by running `npm install -g 'git+ssh://git@github.com:knutties/newman.git#feature/newman-dir'`
+  ```doc
+  Usage: newman [options] [command]
+
+  Options:
+    -v, --version                                               output the version number
+    -h, --help                                                  display help for command
+
+  Commands:
+  dir-add-folder [options] <folder-path>                      Add a folder to directory based Postman collection in the given path
+  dir-add-test [options] <test-path>                          Add a test to directory based Postman collection in the given path
+  dir-create [options] <collection-path>                      Create a directory based Postman collection in the given path
+  dir-export [options] <postman-collection-file>              Convert a Postman collection file into its directory representation
+  dir-export-import-test [options] <postman-collection-file>  Check if an export followed by import results in same collection
+  dir-import [options] <collection-dir>                       Convert a Postman directory representation into a postman collection
+  dir-remove-folder <folder-path>                             Remove test at given path from directory based Postman collection
+  dir-remove-test <test-path>                                 Remove test at given path from directory based Postman collection
+  dir-run [options] <collection-dir>                          Runs the tests in collection-dir, with all the provided options
+  run [options] <collection>                                  Initiate a Postman Collection run from a given URL or path 
+  ```
 - Add the connector credentials to the `connector_auth.toml` / `auth.toml`
-- In terminal, execute:
+- Export the auth file path as an environment variable:
   ```zsh
```

Comment:
Prefer using `shell` instead of `bash` or `zsh`, here and other usages.

```suggestion
  ```shell
```